_rules/presentational-children-no-focusable-content-307n5z.md

```diff
@@ -127,14 +137,54 @@ This element with `menuitemcheckbox` role has a checkbox as a child. Because the
 </ul>
 ```
 
+#### Failed Example 4
+
+This element with the `tab` role contains an `<a>` element.  The `tab` role has [presentational children][].  The `a` element is included in [sequential focus navigation][].  So the element with the `tab` role fails the rule.  (This tablist implementation is non-functional for users.  It's not meant to function - it's only meant to show roles.)
+
+```html
+<ul role="tablist">
+	<li role="tab">
+		<a href="#">Tab 1</a>
+	</li>
+</ul>
+```
+
+#### Failed Example 5
+
+This element with the `img` role contains an `<a>` element.  The `img` role has [presentational children][].  The `<a>` element is included in [sequential focus navigation][].  So the element with the `img` role fails the rule.
+
+```html
+<span role="img" aria-label="some ASCII art">****** This ASCII art ******* <a href="#">contains a link.</a></span>
+```
+
 ### Inapplicable
 
 #### Inapplicable Example 1
 
-This element has a `link` role which does not have [presentational children][].
+The roles that build this semantic table structure (`table` for `<table>`, `row` for `<tr>`, and `cell` for `<td>`) do not have [presentational children][].  So this rule does not apply to them.  (This rule does apply to the `<button>` element within the table.  The `<button>` element passes the rule.)
```

Comment:
I still believe the `button` should go away…
From an implementer point of view, tools will report "Passed" rather than "Inapplicable" on that example. This is not a huge deal as it they are equivalent for consistency, but it still creates some weirdness. 

We could have this example as a Passed Example. It would still fulfil the same goal (show that `table` role and its friends are not applicable) (because tools that apply the rule to them would report a Failed outcome). OTOH, I am not sure that the `button` is adding much value to the example (the link is already a focusable element inside the table, we can have two links and the example is still working as intended).

Reply:
Fair points, I'll discuss one-on-one some time.

Reply:
I agree with @Jym77 

If we want this as an inapplicable example then we should not ask implementations to output "Passed". I agree with either solution proposed: make this a passed example or remove the `button` element.

Reply:
I fixed this a minute ago by removing the button element.